components/omnibox/browser/brave_bookmark_provider.cc

```diff
@@ -43,20 +44,14 @@ void BraveBookmarkProvider::Start(const AutocompleteInput& input,
     auto bump_match =
         base::Contains(lower_contents, lower_text) ||
         base::Contains(base::ToLowerASCII(match.description), lower_text);
-    match.allowed_to_be_default_match = bump_match;
     if (!bump_match) {
       continue;
     }
 
+    match.SetAllowedToBeDefault(input);
```

Comment:
ah, got it. we ignored `prevent_inline_autocomplete()`!

Reply:
nice! 😎👍